src/docfx/docset/Docset.cs

```diff
@@ -224,7 +224,7 @@ Repository GetRepositoryInternal(string fullPath)
             }
         }
 
-        private static IReadOnlyDictionary<string, string> NormalizeRoutes(Dictionary<string, string> routes)
+        private static IReadOnlyDictionary<string, string> NormalizeRoutes(Dictionary<string, string> routes, string sourceBasePath, string siteBasePath)
```

Comment:
where is `sourceBasePath` used?

Reply:
line 239

---
In reply to: [281133469](https://github.com/dotnet/docfx/pull/4481#discussion_r281133469) [](ancestors = 281133469)